legend-engine-protocol-pure/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/packageableElement/store/modelToModel/connection/ModelChainConnection.java

```diff
@@ -22,6 +23,7 @@
 public class ModelChainConnection extends Connection
 {
     public List<String> mappings;
+    public SourceInformation mappingsSourceInformation;
```

Comment:
it really upsets me that we still need to do this 😞 if only we could use mapping pointers

Reply:
yup.